### PR TITLE
Hand 88

### PR DESCRIPTION
**Pull request by yiancar**
## Description
Hand 88 PCB addition

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hand88/readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Hardware Supported: A TKL keyboard with STM32F072CB
+* Hardware Availability: https://www.hand.engineering 
+
+## Instructions
+
+### Build
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hyprkeys_alpha8x:via
```
Comment (Xelus22) [inline keyboards/hand88/readme.md:15]: should this be `make hand88:via` or `make hand88:default`?

Comment (yiancar) [reply keyboards/hand88/readme.md:15]: cheers thanks!

Comment (yiancar) [reply keyboards/hand88/readme.md]: now fixed!

**Comment by yiancar** [comment]
@zvecr anything else obvious here?:)

File: keyboards/hand88/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+BACKLIGHT_ENABLE = no              # Enable keyboard backlight functionality
+BOOTMAGIC_ENABLE = lite            # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes              # Mouse keys
+EXTRAKEY_ENABLE = yes              # Audio control and System control
+CONSOLE_ENABLE = no                # Console for debug
+COMMAND_ENABLE = no                # Commands for debug and configuration
+SLEEP_LED_ENABLE = no              # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes                  # USB Nkey Rollover
+AUDIO_ENABLE = no                  # Audio output on port C6
```
Comment (drashna) [inline keyboards/hand88/rules.mk:20]:
```suggestion
AUDIO_ENABLE = no                  # Audio output
```

**Review by drashna — APPROVED** [review]

**Comment by yiancar** [comment]
Bump:))

**Review by mechmerlin — APPROVED** [review]
LGTM

File: keyboards/hand88/config.h [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+ *
+ * Magic keys are hotkey commands that allow control over firmware functions of
+ * the keyboard. They are best used in combination with the HID Listen program,
+ * found here: https://www.pjrc.com/teensy/hid_listen.html
+ *
+ * The options below allow the magic key functionality to be changed. This is
+ * useful if your keyboard/keypad is missing keys and you want magic key support.
+ *
+ */
+
```
Comment (fauxpark) [inline keyboards/hand88/config.h:89]:
```suggestion
```

File: keyboards/hand88/readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- Plug In
+- Unplug
+- Release Escape
+
+### Flash
+
+- Unplug
+- Hold Escape
+- Plug In
+- Flash using QMK Toolbox or dfu-util (`make hand88:<keymap>:dfu-util`)
```
Comment (fauxpark) [inline keyboards/hand88/readme.md:32]:
```suggestion
- Flash using QMK Toolbox or CLI (`make hand88:<keymap>:flash`)
```

File: keyboards/hand88/readme.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+* Hardware Supported: A TKL keyboard with STM32F072CB
+* Hardware Availability: https://www.hand.engineering 
+
+## Instructions
+
+### Build
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hand88:via
```
Comment (fauxpark) [inline keyboards/hand88/readme.md:15]:
```suggestion
    make hand88:default
```

File: keyboards/hand88/rules.mk [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+BOOTMAGIC_ENABLE = lite            # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes              # Mouse keys
+EXTRAKEY_ENABLE = yes              # Audio control and System control
+CONSOLE_ENABLE = no                # Console for debug
+COMMAND_ENABLE = no                # Commands for debug and configuration
+SLEEP_LED_ENABLE = no              # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes                  # USB Nkey Rollover
+AUDIO_ENABLE = no                  # Audio output
+NO_USB_STARTUP_CHECK = no          # Disable initialization only when usb is plugged in
+RGBLIGHT_ENABLE = no               # Underglow RGB
```
Comment (fauxpark) [inline keyboards/hand88/rules.mk:22]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
NO_USB_STARTUP_CHECK = no   # Disable initialization only when usb is plugged in
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**